agent/fw_drupal8.c

```diff
@@ -534,19 +534,29 @@ NR_PHP_WRAPPER(nr_drupal8_name_the_wt_via_symfony) {
 NR_PHP_WRAPPER_END
 
 void nr_drupal_version() {
-  char* string = "Drupal::VERSION;";
-  zval retval;
-  int result
-      = zend_eval_string(string, &retval, "Retrieve Drupal Version");
-  
+  zval* zval_version = NULL;
+  zend_class_entry* class_entry = NULL;
+
+  class_entry = nr_php_find_class("drupal");
+  if (NULL == class_entry) {
+    nrl_verbosedebug(NRL_INSTRUMENT, "%s: Drupal class not found", __func__);
```

Comment:
Not any Drupal class, the 'Drupal' class:
```suggestion
    nrl_verbosedebug(NRL_INSTRUMENT, "%s: 'Drupal' class not found", __func__);
```